### PR TITLE
feat(proxy_conf): Reworked jinja Template for Zabbix Proxy 5.0 LTS

### PR DESCRIPTION
**Pull request by xenadmin**
### PR progress checklist (to be filled in by reviewers)

- [X] Changes to documentation are appropriate (or tick if not required)
- [X] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [X] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

Reworked jinja Template for Zabbix Proxy 5.0 LTS

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

**Review by hatifnatt — CHANGES_REQUESTED** [review]
Please update commit message according with formulas guidelines
https://github.com/saltstack-formulas/.github/blob/master/CONTRIBUTING.rst#commit-message-formatting

**Comment by xenadmin** [comment]
>Please update commit message according with formulas guidelines

I've read that, and I have no idea that I need to change, sorry. Can you please give me a hint what exactly is missing?

File: zabbix/files/default/etc/zabbix/zabbix_proxy.conf.jinja [hunk 1]
```diff
@@ -143,19 +143,19 @@
 
 {% if zabbix.version_repo|float >= 3.4 -%}
 ### Option: EnableRemoteCommands
-#       Whether remote commands from Zabbix server are allowed.
-#       0 - not allowed
-#       1 - allowed
+#		Whether remote commands from Zabbix server are allowed.
```
[the remark this answers is not in the document]
Comment (xenadmin) [reply zabbix/files/default/etc/zabbix/zabbix_proxy.conf.jinja:146]: Technically not, but I did my best at _exactly_ comparing the original conf from `zabbix-proxy-sqlite3_5.0.37-1+debian11_amd64.deb` with the jinja and I adopted _every_ little change. 
Ps.: If this gets merged, I prepare the next PR with the changes for Zabbix proxy 6.0 LTS.

[the remark this answers is not in the document]
Comment (hatifnatt) [reply zabbix/files/default/etc/zabbix/zabbix_proxy.conf.jinja:146]: Ok, lets update those spaces.

**Review by hatifnatt — APPROVED** [review]
Unfortunately CI is broken but changes looks good for me.

**Pull request merged**